core/classes/Adapter.js

```diff
@@ -200,7 +217,79 @@ module.exports = class Adapter {
       }
     })
 
-    return trx ? query.transacting(trx) : query
+    let result = await (trx ? query.transacting(trx) : query)
+    for (let field of otmPopulate) {
+      await this._oneToModelPopulate(result, field)
+    }
+    for (let field of mtmPopulate) {
+      await this._modelToModelPopulate(result, field.linkerModel, field.initialModel, field.modelToLink)
+    }
+    return result
+  }
+
+  /**
+   * Populates the given models with 1:m relation
+   * @param {Object[]} initialModelArr The array of models to be populated
+   * @param {String} populatingModelIdentity The identity of related model
+   * @private
+   */
+  async _oneToModelPopulate(initialModelArr, populatingModelIdentity) {
+    let idsSet = new Set()
+    for (let model of initialModelArr) {
+      idsSet.add(model[populatingModelIdentity])
+    }
+    const idsArr = [...idsSet]
+
+    const populatingModelsArr = await this.knex(populatingModelIdentity.toLowerCase()).whereIn("id", idsArr)
+    for (let model of initialModelArr) {
+      for (let populatingModel of populatingModelsArr) {
+        if (model[populatingModelIdentity] === populatingModel.id) {
```

Comment:
No, this is very bad. You assume that the attribute of our model should be exactly the same as populating model identity. Imagine the situation, when we have two connections from the `Door` model to Operator model. Let's say they are `createdBy` and `updatedBy` – they show us which operator created the door and which one is the latest who modified it. Since these fields can be named as `operator` both, you won't be able to implement this connection using this algorithm.